beacon-chain/rpc/lookup/blocker.go

```diff
@@ -266,6 +276,27 @@ func (p *BeaconDbBlocker) Blobs(ctx context.Context, id string, indices []int) (
 		}
 	}
 
+	// Convert versioned hashes to indices if provided
+	indices := cfg.Indices
+	if len(cfg.VersionedHashes) > 0 {
+		// Build a map of versioned hash to blob index
+		hashToIndex := make(map[string]int)
+		for i, commitment := range commitments {
+			versionedHash := primitives.ConvertKzgCommitmentToVersionedHash(commitment)
+			hashToIndex[string(versionedHash[:])] = i
+		}
+
+		// Create indices array in the order of requested versioned hashes
```

Comment:
This is not how the spec defines the result

> Blobs are returned as an ordered list matching the order of their corresponding **_KZG commitments_** in the block.

Reply:
whoops good catch